### PR TITLE
fix miss binding for cocostudio-gui change.

### PR DESCRIPTION
**Pull request by WenhaiLin**

### CODE REVIEW

File: tools/tojs/cocos2dx_studio.ini [hunk 1]
```diff
@@ -68,7 +67,8 @@ rename_classes = CCArmature::Armature,
             CCArmatureAnimation::ArmatureAnimation,
             CCSkin::Skin,
             CCBone::Bone,
-            CCArmatureDataManager::ArmatureDataManager
+            CCArmatureDataManager::ArmatureDataManager,
+            TouchGroup::UILayer
```
Comment (dumganhar) [inline tools/tojs/cocos2dx_studio.ini:71]: Why to rename `TouchGroup` to `UILayer`?

Comment (WenhaiLin) [inline tools/tojs/cocos2dx_studio.ini:71]: cocos2d-html5 keep use uilayer,not rename to touchgroup

Comment (dumganhar) [inline tools/tojs/cocos2dx_studio.ini:71]: -_-!!!, all right...

**Comment by dumganhar** [comment]
Thanks.

**Pull request merged**